plugins/modules/nmcli.py

```diff
@@ -1618,8 +1627,12 @@ def connection_options(self, detect_change=False):
         elif self.type == 'team':
             options.update({
                 'team.runner': self.runner,
-                'team.runner-hwaddr-policy': self.runner_hwaddr_policy,
+                'team.runner-hwaddr-policy': self.runner_hwaddr_policy
```

Comment:
Please keep the trailing comma:
```suggestion
                'team.runner-hwaddr-policy': self.runner_hwaddr_policy,
```